Makefile

```diff
@@ -154,7 +153,10 @@ $(BUILD_DIR) $(DATA_DIR) $(DIST_DIR) $(DIST_DATA_DIR):
 	$(MKDIR) $@
 
 test: $(GIT_TEST_DATA_REPO_PATH)
-	$(TESTRUNNER) tests
+	@echo "[run]" > .coveragerc
+	@echo "omit = */invest/ui/*" >> .coveragerc
+	-$(TESTRUNNER) tests/test_validation.py
```

Comment:
Looks like this still just running the tests for validation and not the full suite of model tests.